### PR TITLE
Add rule-properties-order

### PR DESCRIPTION
**Pull request by jeddy3**
I thought I'd make a start on this one.

You can specify either alphabetical or supply your own list of properties.

Ok with you guys?

### CODE REVIEW

File: src/rules/rule-properties-order/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+```css
+a { 
+  -moz-transform: scale(1);
+  -webkit-transform: scale(1);
+  transform: scale(1); 
+}
+```
+
+### `["array", "of", "unprefixed", property", "names"]`
```
Comment (MoOx) [inline src/rules/rule-properties-order/README.md]: how are handled unspecified properties ?

Comment (jeddy3) [inline src/rules/rule-properties-order/README.md]: Oops. Not very well at the moment. Thoughts on the desired behaviour?

Comment (MoOx) [inline src/rules/rule-properties-order/README.md]: Not sure if ignoring them is a good idea. But I guess it's better than nothing for a start :)
Or at the end ? No idea :D

Comment (davidtheclark) [inline src/rules/rule-properties-order/README.md]: I think allowing them at the end is better than allowing them anywhere in the order.

Comment (jeddy3) [inline src/rules/rule-properties-order/README.md]: At the end works for me.

Comment (MoOx) [inline src/rules/rule-properties-order/README.md]: lgtm at the end :)

**Comment by jeddy3** [comment]
I had to refactor the conditionals a make things a little clearer.

Unknown properties must go at the end of rule now.

**Comment by davidtheclark** [comment]
Good rule to start work on. Thanks.

**Pull request merged**